### DISCUSSION TITLE
ZIPKIN WITH DEBEZIUM KAFKA CONNECT

### DISCUSSION THREAD

**Discussion by rayanegouda** [jhc-systems/debezium-connector-ibmi#72] [Q&A]
Hello,
I run the docker-compose.yml.There is a kafka connect debezium/connect image and I added the jar: brave-kafka-interceptor-0.6.0.jar to the path plugin.
As you can see below the brave-kafka-interceptor-0.6.0.jar plugin was loaded by the jvm.
But when we deploy the kafka-connect debezium connector the connector gives me a class not found error.
Having worked on a similar subject would you have any possible resolutions?

<img width="1440" alt="load-jar" src="https://github.com/jhc-systems/debezium-connector-ibmi/assets/10341501/d1bb507c-747b-4a19-9f96-9f14306002d8">
<img width="1275" alt="push-connector" src="https://github.com/jhc-systems/debezium-connector-ibmi/assets/10341501/4cb18152-7b60-479c-b367-e419142b1b63">
<img width="1439" alt="status-push-connector" src="https://github.com/jhc-systems/debezium-connector-ibmi/assets/10341501/733bcbe9-ef31-4c7f-b8a6-b3bf8443a45e">
Le projet est disponilble sur ce lien: 
https://gitlab.com/rayane.gouda/traceproj/-/tree/main/tracabilite?ref_type=heads

Regards,

**Comment by rayanegouda** [comment]
docker compose file: https://gitlab.com/rayane.gouda/traceproj/-/blob/main/tracabilite/docker-compose.yaml?ref_type=heads

**Comment by msillence** [comment]
I suspect you need to add the jar to the main path not simply the plugin path - the plugin path might not be loaded soon enough
